dask_jobqueue/core.py

```diff
@@ -607,6 +625,46 @@ def _new_worker_name(self, worker_number):
             cluster_name=self._name, worker_number=worker_number
         )
 
+    def _get_worker_security(self, security):
+        """Dump temporary parts of the security object into a shared_directory"""
+        if security is not None:
+            copied_on_write = False
+            worker_security_dict = security.get_tls_config_for_role("worker")
+            for key, value in worker_security_dict.items():
+                # dump worker in-memory keys for use in job_script
+                if value is not None and "\n" in value:
+                    if not copied_on_write:
+                        security = copy.copy(security)
+                        copied_on_write = True
```

Comment:
Can't we just copy security object as the beginning and dropt this boolean variable? I do not find it super useful... But maybe I've missed something.

Reply:
This just makes sure that everything is the same if no temp cert is given. Not super useful indeed, just wanted to be safe. I'll remove that